### ISSUE TITLE
Enable record view navigation in the toolbar

### ISSUE THREAD

**Issue by drh-stanford** [sul-dlss/SearchWorks#1499]
Currently blocked by https://github.com/ebsco/edsapi-ruby/issues/48. 

The record view navigation was disabled in #1498 and can be re-enabled by reverting those changes.

**Comment by drh-stanford** [comment]
duplicate of #1471
